src/main/java/com/codegym/shopyy/security/JwtTokenProvider.java

```diff
@@ -0,0 +1,72 @@
+package com.codegym.shopyy.security;
+
+import io.jsonwebtoken.*;
```

Comment:
change to specific-import instead of import .*

Reply:
resolved

Reply:
resolved